### DISCUSSION TITLE
Does "most copied" sort include WearOS?

### DISCUSSION THREAD

**Discussion by Daniel15** [stratumauth/app#1070] [General]
Firstly, thank you for the fantastic app! I've been using it for a long time and really appreciate you working on it.

I usually use Authenticator Pro via my Samsung watch, but I do sometimes use it on my phone.

If I sort by "most copied" or "least copied" in the app, does that count when I view the code on my watch, or is it only when I tap to copy it in the app itself?

What I want to do is find all my most used sites so I can put them in a separate category that I always use, to avoid having to scroll past a bunch of sites that I very rarely need.

**Comment by jamie-mh** [comment]
The sync is one-directional, so viewing the code on your watch does not increment the copy counter. Only copying (when tapped) on the phone does this.
